core/blockchain/validate.go

```diff
@@ -533,20 +544,20 @@ func (b *BlockChain) checkBlockSubsidy(block *types.SerializedBlock, totalFee in
 	// check subsidy
 	transactions := block.Transactions()
 	subsidy := b.subsidyCache.CalcBlockSubsidy(int64(blocks))
-	workAmountOut := int64(transactions[0].Tx.TxOut[0].Amount)
-
-	hasTax := false
-	if b.params.BlockTaxProportion > 0 &&
-		len(b.params.OrganizationPkScript) > 0 {
-		hasTax = true
+	workAmountOut := int64(0)
```

Comment:
The logic change here requires the unit-test coverage to convince the correctness. 
It's difficult to understand the change from `tx[0].output[0].amount` to `tx[0].output[2:].amount`

Reply:
https://github.com/Qitmeer/qitmeer/pull/261/commits/0b232fced328fbc073dfb63676d3392ceb41e2c6